blog/_posts/2016-11-16-team-kindr3d-beyond-tellerrand-16.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Team kindr3d @ Beyond Tellerrand 16
+layout: post
+created_at: Tue Nov 16 2016
+permalink: blog/2016-11-16-team-kindr3d-beyond-tellerland-16
+current: blog
+author: kindr3d
+twitter: TKindr3d
+categories:
+- student-posts
+---
+
+### Team kindr3d @ Beyond Tellerrand 16
+
+
+Team kindr3d were lucky enough to get free tickets for this year [Berlin BTconf](https://beyondtellerrand.com/events/berlin-2016/speakers). We want to say thanks to every member of RGSoC team who fought for us and got us the golden tickets to this unique event which gave us a lot to think about.
+We also met [Nynne](https://twitter.com/nynnest), last year’s alumni and also our supervisor for a week this summer. And we had great conversations and exchanged our experience about our own RGSoC and being a female junior developer.
+
+
+![Don't Panic](https://saintsebastian.github.io/content/images/2016/11/FullSizeRender.jpg)
```

Comment:
To avoid broken images in the post, please add all images to our `img` folder as explained in the [contributing guide](https://github.com/rails-girls-summer-of-code/summer-of-code/blob/gh-pages/CONTRIBUTING.md#images). Please also add image credits to each picture!